main.py

```diff
@@ -10,6 +10,9 @@ def webhook(request):
             try:
                 jsonData = request.get_json()
```

Comment:
In this try section can you print 3 logs, 1 each for different logging level

Reply:
Okay